### ISSUE TITLE
ENOENT error on run since 4.1.1

### ISSUE THREAD

**Issue by JakeRG** [tuananh/camaro#85]
**Describe the bug**
After updating to camaro 4.1.1, program does not run and instead crashes with an ENOENT error. Deleting node_modules and package-lock.json does not help, but downgrade to 4.1.0 and running npm install again resolves the issue.

**Error:**
```
{ Error: ENOENT: no such file or directory, open '<<PROJECT LOCATION>>\node_modules\camaro\dist\camaro.wasm'
    at Object.openSync (fs.js:438:3)
    at Object.readFileSync (fs.js:343:35)
    at w (<<PROJECT LOCATION>>\node_modules\camaro\dist\camaro.js:9:121)
    at x (<<PROJECT LOCATION>>\node_modules\camaro\dist\camaro.js:9:178)
    at Ea (<<PROJECT LOCATION>>\node_modules\camaro\dist\camaro.js:17:305)
    at Fa.u.function.fetch.credentials (<<PROJECT LOCATION>>\node_modules\camaro\dist\camaro.js:18:85)
    at new Promise (<anonymous>)
    at Fa (<<PROJECT LOCATION>>\node_modules\camaro\dist\camaro.js:18:59)
    at d (<<PROJECT LOCATION>>\node_modules\camaro\dist\camaro.js:19:220)
    at <<PROJECT LOCATION>>\node_modules\camaro\dist\camaro.js:20:193
    at Ga (<<PROJECT LOCATION>>\node_modules\camaro\dist\camaro.js:20:412)
    at Object.e.asm (<<PROJECT LOCATION>>\node_modules\camaro\dist\camaro.js:21:152)
    at <<PROJECT LOCATION>>\node_modules\camaro\dist\camaro.js:36:10
    at Object.<anonymous> (<<PROJECT LOCATION>>\node_modules\camaro\index.js:12:18)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
  errno: -4058,
  syscall: 'open',
  code: 'ENOENT',
  path:
   '<<PROJECT LOCATION>>\\node_modules\\camaro\\dist\\camaro.wasm' }
{ Error: ENOENT: no such file or directory, open '<<PROJECT LOCATION>>\node_modules\camaro\dist\camaro.wasm'
    at Object.openSync (fs.js:438:3)
    at Object.readFileSync (fs.js:343:35)
    at w (<<PROJECT LOCATION>>\node_modules\camaro\dist\camaro.js:9:121)
    at x (<<PROJECT LOCATION>>\node_modules\camaro\dist\camaro.js:9:178)
    at Ea (<<PROJECT LOCATION>>\node_modules\camaro\dist\camaro.js:17:305)
    at Fa.u.function.fetch.credentials (<<PROJECT LOCATION>>\node_modules\camaro\dist\camaro.js:18:85)
    at new Promise (<anonymous>)
    at Fa (<<PROJECT LOCATION>>\node_modules\camaro\dist\camaro.js:18:59)
    at d (<<PROJECT LOCATION>>\node_modules\camaro\dist\camaro.js:19:220)
    at <<PROJECT LOCATION>>\node_modules\camaro\dist\camaro.js:20:193
    at Ga (<<PROJECT LOCATION>>\node_modules\camaro\dist\camaro.js:20:412)
    at Object.e.asm (<<PROJECT LOCATION>>\node_modules\camaro\dist\camaro.js:21:152)
    at <<PROJECT LOCATION>>\node_modules\camaro\dist\camaro.js:36:10
    at Object.<anonymous> (<<PROJECT LOCATION>>\node_modules\camaro\index.js:12:18)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
  errno: -4058,
  syscall: 'open',
  code: 'ENOENT',
  path:
   '<<PROJECT LOCATION>>\\node_modules\\camaro\\dist\\camaro.wasm' }
```

- camaro version: 4.1.1
- Node version: 10.15.3
- Operating system: Windows 10

**Comment by JakeRG** [comment]
Very sorry about that, misclicked. But now that I'm here anyway; thanks for writing camaro, I love it!

**Comment by tuananh** [comment]
@JakeRG yeah my bad. i forgot to whitelist the .wasm file. will publish a fix soon.

**Comment by tuananh** [comment]
Should be fixed now in 4.1.2.

**Comment by alsatianio** [comment]
@tuananh I've moved camaro away from web and into Node, and I'm getting the same as the above error right now. See below:

```
Error: ENOENT: no such file or directory, open '<<PROJECT_DIR>>/node_modules/camaro/dist/camaro.wasm'
    at Object.openSync (fs.js:461:3)
    at Object.readFileSync (fs.js:364:35)
    at w (<<PROJECT_DIR>>/node_modules/camaro/dist/camaro.js:9:121)
    at x (<<PROJECT_DIR>>/node_modules/camaro/dist/camaro.js:9:178)
    at Ea (<<PROJECT_DIR>>/node_modules/camaro/dist/camaro.js:17:305)
    at Fa.fetch.credentials (<<PROJECT_DIR>>/node_modules/camaro/dist/camaro.js:18:85)
    at new Promise (<anonymous>)
    at Fa (<<PROJECT_DIR>>/node_modules/camaro/dist/camaro.js:18:59)
    at d (<<PROJECT_DIR>>/node_modules/camaro/dist/camaro.js:19:220)
    at <<PROJECT_DIR>>/node_modules/camaro/dist/camaro.js:20:193 {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: '<<PROJECT_DIR>>/node_modules/camaro/dist/camaro.wasm'
}
```
